### ISSUE TITLE
Communcation between UI and API not working when using Self-Signed certificates in Openshift

### ISSUE THREAD

**Issue by pszuster** [Apicurio/apicurio-studio#340]
The UI can't communicate with the other pods when deployed on Openshift using self-signed certificates. The workaround is to open their URLs and accept their certificates before using the UI.

**Comment by EricWittmann** [comment]
Yep - this is definitely true.  I'm open to a solution if you have one.  The problem is that the browser is making direct API calls to the two back-end pods (one pod for the REST API and one for the editor Web Socket).  Obviously when the routes are SSL and the certs are invalid or self-signed, the browser will NOT connect without first asking the user for confirmation.  I know of no way to fix this.  

You can use HTTP routes instead of HTTPS (obviously).  Or you can fix the SSL certs so the browser doesn't complain.  Or use the workaround you mention above.
